addons/one-click-fluffyscratch-login/addon.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "One Click FluffyScratch Login",
+  "description": "Makes logging in to sites that use FluffyScratch F A S T.",
+  "credits": [
+    {
+      "name": "easrng",
+      "link": "https://easrng.net/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/413751680", "https://scratch.mit.edu/projects/413751680/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "hideuntilscript.css",
+      "matches": ["https://scratch.mit.edu/projects/413751680", "https://scratch.mit.edu/projects/413751680/*"]
+    }
+  ],
+  "tags": [],
+  "enabledByDefault": true
```

Comment:
why

Reply:
An enabled by default Easter egg would be interesting.

Reply:
because it makes testing easier